### ISSUE TITLE
Accessibility and usability improvements

### ISSUE THREAD

**Issue by jokd** [origo-map/origo#1211]
We shall test and rework Origo to better comply with guidelines regarding usability such as WCAG 2.1 (https://www.w3.org/TR/WCAG21/).

It is good but can be even better. #667 is a nice improvement.

**Comment by johnnyblasta** [comment]
I looked into some buttons that is missing aria-label and will be coming with a PR for #1297
